install_files/ansible-base/roles/backup/tasks/restore.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: copy restore script to app server
+  copy: src=0.3_restore.py dest=/tmp/ owner=root mode=770
```

Comment:
For consistency, use the multiline YAML syntax here, as in the `file:` task below:

```
- name: Copy restore file to App Server.
  copy:
    src: "{{ restore_file }}"
    dest: /tmp/{{ restore_file }}
```